### PR TITLE
[DOCS-3170] Add note for v4 cutoff

### PR DESCRIPTION
**Pull request by jrodewig**
### Notes
Rel: https://faunadb.atlassian.net/browse/DOCS-3170

### CODE REVIEW

**Review by ecooper — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -6,13 +6,16 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 > [!WARNING]
->  Fauna is decomissioning FQL v4 on June 30, 2025. See the [v4 end of life (EOL)
+>  Fauna is decomissioning FQL v4 on June 30, 2025.
+>
+> This driver is not compatible with FQL v10, the latest version. Fauna accounts
+> created after Month DD, YYYY must use FQL v10. Ensure you migrate existing
```
Comment (ecooper) [inline README.md]: I'm assuming we'll replace this with the correct date prior to merging?

Comment (jrodewig) [reply README.md]: Yep — This is just a placeholder until we have a firm date. Thanks!

File: README.md [hunk 2]
```diff
@@ -6,13 +6,16 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 > [!WARNING]
->  Fauna is decomissioning FQL v4 on June 30, 2025. See the [v4 end of life (EOL)
+>  Fauna is decomissioning FQL v4 on June 30, 2025.
```
Comment (mryan-fauna) [inline README.md]: decommissioning

File: README.md [hunk 3]
```diff
@@ -6,13 +6,16 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 > [!WARNING]
->  Fauna is decomissioning FQL v4 on June 30, 2025. See the [v4 end of life (EOL)
+>  Fauna is decommissioning FQL v4 on June 30, 2025.
```
Comment (mryan-fauna) [inline README.md:9]: decommissioning

**Review by mryan-fauna — APPROVED** [review]

**Pull request merged**